src/util/privkey.rs

```diff
@@ -110,13 +109,19 @@ impl Display for Privkey {
         };
         fmt.write_str(&privkey)
     }
-}
 
-impl FromStr for Privkey {
-    type Err = encode::Error;
+    /// Get WIF encoding of this private key.
+    #[inline]
+    pub fn to_wif(&self) -> String {
```

Comment:
This is how `ToString` is implemented.

Reply:
Can you just call `self.to_string()` directly?